_includes/docs/0.4.x/gettingstarted.html

```diff
@@ -39,7 +39,7 @@
     There are several ways you can get the MEAN.JS boilerplate:
   </p>
   <h3>Yo Generator</h3>
-  <p>The generator is still in development. See the <a href="https://github.com/meanjs/generator-meanjs/tree/0.4-dev">Generator Repo</a> for information on how to use the development version.
+  <p>The generator is still in development. See the <a href="https://github.com/meanjs/generator-meanjs">Generator Repo</a> for information on how to use the development version.
```

Comment:
Let's change this to something along the lines of the generator is released but we are adding back features from 0.3.3 generator as we convert the code